### ISSUE TITLE
Server Side Events Compatability 

### ISSUE THREAD

**Issue by torkelo** [intelsdi-x/snap#766]
Not sure if I am doing something wrong but I fail to watch tasks using Server Side Events browser standard  (https://www.w3.org/TR/eventsource/#parsing-an-event-stream) 

The Content-Type is "text/event-stream" which says is should be. But the EventSource onmessage is never called if you try to watch a task from a browser. 

```javascript
var source = new EventSource(watchUrl);
source.onmessage = function(evt) {
  console.log('event message', evt);
};
source.onerror = function(evt) {
  console.log('event error', evt);
};
```

According to the Server Side Events standard each event needs to be formatted like this
```
data: <data>;<new_line>
```

**Comment by lynxbat** [comment]
@torkelo could you share the `EventSource` lib you are using above?

We probably missed making the stream SSE compliant.

**Comment by torkelo** [comment]
@lynxbat it's a web / browser standard lib, included in the javascript API in most browsers
https://developer.mozilla.org/en-US/docs/Web/API/EventSource

**Comment by lynxbat** [comment]
Thanks @torkelo 

We will take a look at this right away.

**Comment by lynxbat** [comment]
@torkelo see above

**Comment by ConnorDoyle** [comment]
Fixed by #772

**Comment by torkelo** [comment]
awesome! tested it earlier today, works great now. Thanks for moving so quickly on this!
